erpnext/stock/doctype/batch/batch.py

```diff
@@ -110,11 +110,15 @@ def after_delete(self):
 
 	def validate(self):
 		self.item_has_batch_enabled()
+		self.set_batchwise_valuation()
```

Comment:
so if anyone disables/enables an old batch or adds description to it, it will be set to 1 ?

Reply:
it's "set only once" so won't change it but throw an error. 🥲 

Nice catch, will fix separately.

Reply:
<img width="816" alt="Screenshot 2022-02-21 at 6 12 09 PM" src="https://user-images.githubusercontent.com/9079960/154957321-772180c8-a4a6-4f9e-a043-43450c4ae6a3.png">


can confirm locally

Reply:
https://github.com/frappe/erpnext/pull/29916